### PR TITLE
CLOUD-1946: add power mode logs to influx writes

### PR DESCRIPTION
**Pull request by SMMustafa1**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/outputs/proto/proto.go [hunk 1]
```diff
@@ -405,6 +405,12 @@ func (f *Proto) Write(metrics []telegraf.Metric) error {
 				return errors.Wrap(err, "build glog")
 			}
 			influx.Glog = append(influx.Glog, &m)
+		case "powermode":
```
Comment (anycmon) [inline plugins/outputs/proto/proto.go:408]:
```suggestion
		case "power_mode":
```

File: plugins/outputs/proto/proto.go [hunk 2]
```diff
@@ -405,6 +405,12 @@ func (f *Proto) Write(metrics []telegraf.Metric) error {
...
+			m := PowerMode{}
+			if err := json.Unmarshal(b, &m); err != nil {
+				return errors.Wrap(err, "build powermode")
```
Comment (anycmon) [inline plugins/outputs/proto/proto.go:411]:
```suggestion
				return errors.Wrap(err, "build power_mode")
```

**Review by anycmon — CHANGES_REQUESTED** [review]

File: plugins/outputs/proto/proto.go (hunk 1 above)
Comment (SMMustafa1) [reply plugins/outputs/proto/proto.go:408]: fixed

File: plugins/outputs/proto/proto.go (hunk 2 above)
Comment (SMMustafa1) [reply plugins/outputs/proto/proto.go:411]: fixed

**Review by anycmon — APPROVED** [review]
LGTM

**Pull request merged**